nrf_security/doc/driver_config.rst

```diff
@@ -0,0 +1,344 @@
++----------------+-----------------------------------------------+
+| XTS            | :kconfig:`CONFIG_PSA_WANT_ALG_XTS`            |
++----------------+-----------------------------------------------+
+
+
+AES cipher driver configuration
+===============================
+
+The Kconfig variables in the following table can be used for fine-grained control over which drivers provides AES cipher support:
+
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+| Cipher mode    | nrf_cc3xx driver support                              | nrf_oberon driver support                              |
++================+=======================================================+========================================================+
+| ECB_NO_PADDING | :kconfig:`PSA_CRYPTO_DRIVER_ALG_ECB_NO_PADDING_CC3XX` | :kconfig:`PSA_CRYPTO_DRIVER_ALG_ECB_NO_PADDING_CC3XX`  |
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+| CBC_NO_PADDING | :kconfig:`PSA_CRYPTO_DRIVER_ALG_CBC_NO_PADDING_CC3XX` | :kconfig:`PSA_CRYPTO_DRIVER_ALG_CBC_NO_PADDING_OBERON` |
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+| CBC_PKCS7      | :kconfig:`PSA_CRYPTO_DRIVER_ALG_CBC_PKCS7_CC3XX`      | Not supported                                          |
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+| CFB            | :kconfig:`PSA_CRYPTO_DRIVER_ALG_CFB_CC3XX`            | Not supported                                          |
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+| CTR            | :kconfig:`PSA_CRYPTO_DRIVER_ALG_CTR_CC3XX`            | :kconfig:`PSA_CRYPTO_DRIVER_ALG_CTR_OBERON`            |
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+| OFB            | Not supported                                         | Not supported                                          |
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+| XTS            | Not supported                                         | Not supported                                          |
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+
+.. note::
+   * If an AES cipher mode is enabled and no PSA driver enables or supports it, then builtin Mbed TLS support is enabled and used.
+   * The :ref:`nrf_security_drivers_cc3xx` is limited to AES key sizes of 128 bits on devices with Arm CryptoCell cc310.
```

Comment:
Maybe also mention that otherwise there are two more key sizes supported, unless that is documented elsewhere.

Reply:
The same applies elsewhere

Reply:
Not handled in this PR, to be considered for later improvements